.github/workflows/e2e.yaml

```diff
@@ -7,6 +7,7 @@ on:
   pull_request:
     branches:
       - main
+  workflow_dispatch:
```

Comment:
Why do we need `workflow_dispatch`?

Reply:
Can we use this button?
![image](https://github.com/containers/youki/assets/13010913/b7b1a63c-fd75-4aa4-9bd9-b6459c64ddb2)

Reply:
You can, but you have to create a PR upfront with all CI triggered. And this only do so with already ran CI. I can't just push to a branch (without creating a PR) and trigger 1 specific CI run to test things quickly. What is your main concern for adding this? I can remove it from this PR if you have no issue with the rest, so this one is not blocked.

Reply:
> What is your main concern for adding this?

The debugging of CI in the upstream repositories is to occur. It should do in each fork branch.

Reply:
I agree that the debugging should be done in the forked branch. However, I was not able to use the CI in the forked repo unless the workflow yaml specify that the the `workflow_dispatch` trigger. I can't trigger the CI in the forked repo unless the workflow yaml file in the main branch contains that trigger. I can change the branch matching rule to trigger on every push in the forked repo on my own branch, but I can't add a new trigger without merging this into main.